slicer/slicerrc.py

```diff
@@ -0,0 +1,269 @@
+# Import required libraries
+import numpy as np
+import qt
+import os
+from glob import glob
+import json
+
+# Defining path
+FILE = "/root/buffer.txt"
+ANNOTATIONS_DIR = "/root/annotations"
+
+already_updated = [""]  # array to check if the file requested is same or not
+
+# This function flips inverted dicom images
+def flip_dicoms():
+    sliceNode = slicer.app.layoutManager().sliceWidget("Red").mrmlSliceNode()
+    sliceToRas = sliceNode.GetSliceToRAS()
+    transform = vtk.vtkTransform()
+    transform.SetMatrix(sliceToRas)
+    transform.RotateZ(180)
+    sliceToRas.DeepCopy(transform.GetMatrix())
+    sliceNode.UpdateMatrices()
+
+
+# This function loads a brain volume node on the 3dslicer window
+def brain_window(volumenode):
+    displaynode = volumenode.GetDisplayNode()
+    displaynode.AutoWindowLevelOff()
+    displaynode.SetWindowLevel(100, 50)
+
+
+# This function loads files from given folder path
+def load_dicom_volume_from_folder(path):
```

Comment:
Use the following format for documentation: https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html